lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -32,12 +32,13 @@ public function index() {//$parameters) {
   /**
    * Display the specified resource.
    *
-   * @param string $id Resource id.
+   * @param string $id Signup id.
    * @return array
    */
   public function show($id) {
     try {
       $signup = Signup::get($id);
+      $signup = $signup[0];
```

Comment:
I don't think you need this... you remove it from the array and then wrap back in array for the services function below and then pop out from array.